data/test/news/articles/c0000000003o.json

```diff
@@ -88,39 +85,29 @@
             "locator": "urn:bbc:pips:pid:p067gq6h",
             "blocks": [
               {
-                "type": "rawVideo",
-                "model": {
-                  "guidance": null,
-                  "isLive": false,
-                  "duration": "PT1M34S",
-                  "allowOffsiteEmbedding": false,
-                  "locator": "urn:bbc:pips:pid:p067gq6h"
-                }
-              },
-              {
-                "type": "altText",
-                "model": {
-                  "blocks": [
+                "type" : "aresMedia",
+                "model" : {
+                  "id" : "p067gq6h",
+                  "subType" : "clip",
+                  "format" : "audio_video",
+                  "title" : "Mark and Isla Royal Wedding",
+                  "synopses" : {
+                    "short" : "Stop the press! It’s a Mark and Isla Royal Wedding special…",
+                    "medium" : null,
+                    "long" : null
+                  },
+                  "imageUrl" : "ichef.bbci.co.uk/images/ic/$recipe/p067grwg.jpg",
```

Comment:
We have asked for these fields, however:
1) They aren't currently available in MPV
2) For clips, this is because Castaway doesn't send them to MPV
3) For episodes, this is because Nitro doesn't make them available in the episode metadata

We will have to assume a 16:9 ratio for any video that isn't explicitly portrait, in which case we would assume 9:16.

I hope that we can get this changed, but at the moment that's the limitation of the upstream systems.

Reply:
Would the assumptions not happen in Ares rather than Articles? Seems to me that Optimo and Articles should be as dumb as possible - one serialises & puts into content store, the other blindly renders - the data mapping happens at the Ares layer.

Are we able to request Ares falls back to these defaults? 

```yaml
image:
  height: 360
  width: 640
  alt: null 
```

Reply:
I'd be dubious of this aproach.  Ares needs to be agnostic of who's receiving the data, and other consumers might have no need for this, and if it sees these sizes might default an image to a size that's not appropriate.

Reply:
"We will have to assume a 16:9 ratio for any video that isn't explicitly portrait, in which case we would assume 9:16."

Do we have a JSON example of a video that is explicitly portrait?

Reply:
Example - http://cookbook.tools.bbc.co.uk/vertical

Reply:
@jimjohnsonrollings I still think a lack of alt text is a big problem, putting an image on a page without alt text breaks our accessibility guidelines - unless the image is going to be hidden to screen readers

Reply:
I would have thought lack of alt text is ok, provided the video has a title/caption 🤔 
Thanks for the cookbook recipe 👍 shouldn't we have a 9:16 example in the data in this PR?

Reply:
Scenarios where we would show the image and not the video:
- No JS: image is used as a background to a text messsage saying 'no javascript no video' (exact wording TBC)
- SMP fails to load: image is used as a background to a text message saying 'not working'
- Region restriction: image is used as a background to a text message saying 'you aren't allowed to watch this'
- Media is not yet available: image is used as a background to a text message saying 'ain't started yet'
- Media has ceased to be available: image is used as a background to a text message saying 'sorry it's gone'

In all of these cases the image is irrelevant to screen readers, as the message is the important thing and the image is just decoration, so we agreed with UX that alt text was not required.

Reply:
On 9:16 - I can see if someone can create me one on Test that we can use.